entry_tasks/46_calculator/main.py

```diff
@@ -0,0 +1,18 @@
+if __name__ == "__main__":
+    a = int(input('Enter first number: '))
+    b = int(input('Enter second number: '))
+    operation = input('''
+    Enter operator:
+    + for "addition"
+    > for "greater than"
+    or another operator
+    ''')
+
+if operation == '+':
```

Comment:
this if else block should also be within main

Reply:
variables should be defined and used within proper scope:

if a > 10:
    # if scope
    x = 5
    print(x)
else:
    # else scope
    print(x)

if a would be <= 10 you'll see an error:
`NameError: name 'x' is not defined`

otherwise you'll see `5`  (without any error)

So it's tricky, python won't read if or else block if there is no such need and won't notify you that variable is actually may be not defined. But, PyCharm highlights it, put mouse on a, b below